satellite-placement/README.rst

```diff
@@ -0,0 +1,63 @@
+Satellite Placement Demo
+========================
+Suppose you have a set of ``N`` satellites and ``k`` targets on Earth that you
+want to observe. Each of your satellites has varying capabilities for Earth
+observation; in particular, the amount of ground that they can observe for a
+set amount of time is different. Since there are ``k`` targets, you would like
+to have ``k`` constellations to monitor said targets. How do you group your
+satellites into ``k`` constellations such that the average coverage of each
+constellation is maximized? This is the question that we will be addressing in
+this demo!
+
+Note: in this demo we are assuming that ``N`` is a multiple of ``k``.
+
+Usage
+-----
+To run the demo,
+::
+  python satellite.py
+
+It will print out a set of satellite constellations.
+
+Code Overview
+-------------
+The idea is to consider all possible combinations of satellites, eliminate
+constellations with particularly low coverage, and encourage certain
+properties in our final solution. Namely, we want to encourage:
+
+* Constellations that have better coverage
+* Satellites to only join *one* constellation
+* A specific number of constellations in our final solution (i.e. encourage the
+  solution to have ``k`` constellations)
+
+Code Specifics
+--------------
+
+* The ``score_threshold`` - used to determine bad constellations - was
+  assigned an arbitrarily picked number
+* In ``bqm.add_variable(frozenset(constellation), -score)``, we are using
+  ``frozenset(constellation)`` rather than simply ``constellation`` because
```

Comment:
I think this part might be unclear to users. Perhaps consider separating the technical reason---BQM.add_variable(v, ...) requires that v be a valid dict key---from the algorithmic reason (the use of sets of satellites as variables with its accompanying combinatoric considerations)

Reply:
For this case, I have added more clarification and explained how we end up with frozenset. Essentially, what I had originally wanted to get across was "Why frozenset? It's because it's a set and it's immutable".